content/en/docs/210_reading/212_ImageZoom.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Zooming on images"
+description: ""
+draft: false
+weight: 213
+toc: false
+---
+Since 2.3, Thorium has an image zoom feature (epub only).
+
+To zoom on an image you can select the image (a magenta focus outline will appear around it) and activate zoom by clicking with the mouse or tapping it with the finger. Access to image zoom via keyboard is currently not possible.
+
+If the image is also a link or if the book your read is in a fixed layout format, you will need to hit the <kbd>Shift</kbd> key in order to enter the image zoom feature.
+
+To zoom in or out, tab to the `+` or `-` buttons and hit <kbd>enter</kbd>, use mouse wheel, scroll up and down or pinch and stretch trackpad or onscreen touch gestures.
+
+Once the image is zoomed you can move vertically and horizontally with <kbd>&larr;</kbd>, <kbd>&rarr;</kbd>, <kbd>&uarr;</kbd> and <kbd>&darr;</kbd>, by moving mouse while maintaining clic, with usual trackpad gestures or with finger moves on tactile screens.
```

Comment:
"while maintaining clic" is unclear (and I cannot move using the mouse on a Mac, even with mouse clic, or any supplemental key).